README.md

```diff
@@ -1,2 +1,115 @@
+
+#### kellerkinder:shopware:opcache
+This will reset the OPCache using `cachetool.phar` and the `fcgi_sockets` specified in the `deploy.php`
+
+#### kellerkinder:shopware:plugins
+This will (re)install `K10rDeployment` which gives us some useful commands for the deployment process. Afterwards it deactivates `SwagUpdate` since we do not want anyone to install an update via backend.
+And most importantly it installs and updates all plugins specified in the `plugins` variable in `deploy.php`.
+
+This task should also be used to set plugin configurations that should be the same on all stages like this:
+```
+run("cd {{shopware_public_path}} && {{bin/php}} bin/console sw:plugin:config:set SwagImportExport useCommaDecimal 0");
+```
+
+#### kellerkinder:shopware:config
+This step will import and update the snippets defined via `K10rProject`. It will also be used to set configurations that should be the same in all stages. Since this is highly individual there are no configs set in this basic configuration.
+Feel free to add configuration settings like this:
+```
+run("cd {{shopware_public_path}} && {{bin/php}} bin/console k10r:config:set disableShopwareStatistics 1");
+```
+This would disable the internal shopware statistics. For more information on how to set configurations check [K10rDeployment](https://github.com/kellerkinderDE/K10rDeployment).
+
+
+
+#### kellerkinder:shopware:cache'
+Clears the cache using `sw:cache:clear` and compiles the themes afterwards. Can also be used to warm up the cache which is not activated by default. See code comments.
+
+### kellerkinder:shopware:production and kellerkinder:shopware:staging
+These two commands are used to set configurations that differ between the stages.
+
+The staging task will for example install and activate `K10rStaging` which will show a notice that the user is in a staging environment and catch mails.
+
+This task also sets the shop name and title to `DEV!!!` via `k10r:store:update`.
+
+These tasks are usally used to set plugin configurations differing between stage and production, e.g. PayPal in production or sandbox mode. Or to deactivate a tracking plugin that will only be used production.
+
+
+## Directories on the server
+Within your deploy path deployer will create several directories:
+
+* `.dep`: Information about the releases
+* `current`: Symlink that will always be linked to the latest release.
+* `releases`: This directory contains the last 10 releases. Configure how many releases are kept via `keep_releases` in `deploy-shopware-php`. 
+* `shared/media`: Place your media files here. deployer will symlink them into the shopware root directory across all releases.
+* `shared/files`: Same as above but for the document and download files.
+* `shared/web`: Place your `config.php` here. Also a good place for the `.htaccess` if it's not identical to the one shippied with shopware.
+
+The shared directories and files are configured in `deploy-shopware.php`.
```

Comment:
Maybe add a sentence, that they can be expanded via `add()` in `deploy.php`.

Reply:
Fixed